pkg/utils/globalTypes/health/health.go

```diff
@@ -0,0 +1,33 @@
+package health
+
+type HealthInfo struct {
+	StrongLevel    int // defined as the lowest HP value for a given HP level
+	HealthyLevel   int
+	WeakLevel      int
+	WidthStrong    float64 // width of every level
+	WidthHealthy   float64
+	WidthWeak      float64
+	WidthCritical  float64
+	FoodReqStrong  float64 // food requirement to stay at a given level
+	FoodReqHealthy float64
+	FoodReqWeak    float64
+	FoodReqCToW    float64
+	MaxDayCritical int // maximum number of day at critical level
+}
+
+func NewHealthInfo(StrongLevel, HealthyLevel, WeakLevel int, FoodReqStrong, FoodReqHealthy, FoodReqWeak, FoodReqCToW float64, MaxDayCritical int) *HealthInfo {
+	return &HealthInfo{
+		StrongLevel:    StrongLevel,
+		HealthyLevel:   HealthyLevel,
+		WeakLevel:      WeakLevel,
+		WidthStrong:    float64(100 - StrongLevel),
```

Comment:
I don't think so. If I understand correctly, `agentHP` is the initial HP value of every agent. We can choose to set this initial value to the maximal HP (100), but we can also choose to initialize it with any other value (for example every agent critical at the beginning of the simulation). This has therefore to be an independent variable in my opinion

Reply:
Nothing decisive to add but I was under the impression we would never vary initial HP of an agent in which case I agree with @jzzheng22

Reply:
I think it could be interesting to try different initial conditions for the agent health, and would therefore keep it as a parameter. A society that begins with only "strong" agents has more time to discuss treaties etc in comparison to a society beginning with agent that are already in "critical" state from day 1
Also, this parameter is quite convenient to quickly test different scenarios when implementing a new feature: in my case, I could directly start the simulation at the critical level to check if my specific changes were correctly implemented for example

Reply:
I'm happy to keep `initialHP` and `maxHP` as separate variables, but in either case, 100 should not be there: it should be either `initialHP` (i.e. `agentHP`) or `maxHP`. 100 is a magic number that just appears

Reply:
I think that 100 as a value of `maxHP` will actually not change. I thought we agreed on that earlier but I might be wrong. I would say we can now set 100 as a hard-coded maximal HP value, I don't really see for which purpose we would play with this value in the future

Reply:
I agree with @jzzheng22. instead of writing '100' here, we should probably code it as 'maxHP'. It kind of seems that 100 is a random magic number that appeared out of nowhere, but writing 'maxHP' would make it easier to see how widthStrong is being calculated (i.e. maxHP - StrongLevel).

Reply:
Done, MaxHP is now a variable of the struct HealthInfo. Its value is set at the beginning of the simulation in main.go.